modules/julia/samples/chessboard_corners.jl

```diff
@@ -2,9 +2,8 @@ using OpenCV
 
 const cv = OpenCV
 
-
-# chess1.png is at https://raw.githubusercontent.com/opencv/opencv_extra/master/testdata/cv/cameracalibration/chess1.png
-img = cv.imread("chess1.png",cv.IMREAD_GRAYSCALE)
+file = download("https://raw.githubusercontent.com/opencv/opencv_extra/master/testdata/cv/cameracalibration/chess1.png")
```

Comment:
In general samples should not force downloading of external content.
We should have pre-check if file already exists in current working directory (e.g., downloaded by user before)

Reply:
Honestly, the best way would be to use an Artifact. That would take care of everyone's needs.